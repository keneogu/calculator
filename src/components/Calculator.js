import React from 'react';
import App from './App';

const Calculator = () => (
  <div className="homepage">
    <h2 className="home-head">Compute your problems here.</h2>
    <div>
      <App />
    </div>
  </div>
);

export default Calculator;
