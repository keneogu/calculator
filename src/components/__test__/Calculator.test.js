import renderer from 'react-test-renderer';
import React from 'react';
import Calculator from '../Calculator';

it('matches the Calculator snapshot', () => {
  const tree = renderer.create(<Calculator />).toJSON();
  expect(tree).toMatchSnapshot();
});
