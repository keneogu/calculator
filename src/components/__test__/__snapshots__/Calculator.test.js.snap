// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the Calculator snapshot 1`] = `
<div
  className="homepage"
>
  <h2
    className="home-head"
  >
    Compute your problems here.
  </h2>
  <div>
    <div
      className="contain"
    >
      <div
        className="display"
      >
        <h1
          className="display-head"
          data-testid="display"
        >
          0  
        </h1>
      </div>
      <div
        className="btn-panel"
        data-testid="button-panel"
      >
        <div
          className="inner-panel"
        >
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            AC
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            +/-
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            %
          </button>
          <button
            className="btn secondary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            รท
          </button>
        </div>
        <div
          className="inner-panel"
        >
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            7
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            8
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            9
          </button>
          <button
            className="btn secondary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            x
          </button>
        </div>
        <div
          className="inner-panel"
        >
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            4
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            5
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            6
          </button>
          <button
            className="btn secondary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            -
          </button>
        </div>
        <div
          className="inner-panel"
        >
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            1
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            2
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            3
          </button>
          <button
            className="btn secondary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            +
          </button>
        </div>
        <div
          className="inner-panel"
        >
          <button
            className="btn primary wide"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            0
          </button>
          <button
            className="btn primary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            .
          </button>
          <button
            className="btn secondary"
            data-testid="button"
            onClick={[Function]}
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
